### PR TITLE
rename timeout to maxRuntime

### PR DESCRIPTION
**Pull request by ObiWahn**
trivial rename

### CODE REVIEW

**Comment by ObiWahn** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/7410/

File: CHANGELOG [hunk 1]
```diff
@@ -92,8 +92,8 @@ devel
   engine, but comes with a big performance penalty as all collections will be locked
   exclusively for writes.
 
-* Added new timeout option for AQL queries. If a query does not finish execution
-  within the given time in seconds it will be killed.
+* Added new maxRuntime option for queries. If a query does not finish execution within
+  the given time in second it will be killed.
```
Comment (jsteemann) [inline CHANGELOG]:
```suggestion
  the given time (in seconds) it will be killed.
```

**Comment by jsteemann** [comment]
Tests blue

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/Documentation/job/arangodb-ANY-examples/458/

**Pull request merged**